src/data/emacs-mode/agda-input.el

```diff
@@ -805,13 +852,15 @@ order for the change to take effect."
 
   ;; Musical symbols.
 
-  ("note" . ,(agda-input-to-string-list "♩♪♫♬"))
-  ("b"    . ("♭"))
-  ("#"    . ("♯"))
+  ("note"    . ,(agda-input-to-string-list "♩♪♫♬"))
+  ("b"       . ("♭"))
+  ("#"       . ("♯"))
+  ("bb"      . ("𝄫"))
+  ("##"      . ("𝄪"))
 
   ;; Other punctuation and symbols.
 
-  ("\\"         . ("\\"))
+  ("\\"         . ("＼"))
```

Comment:
Why this?  How we are supposed to type an ordinary backslash now?

I am going to revert this line...

Reply:
This was definitely a mistake. It is so long ago now that I have forgotten any possible rationale I had for making this change.